cmd/proxy/coordinator.go

```diff
@@ -114,6 +114,8 @@ func (c *Coordinator) DoScrape(ctx context.Context, r *http.Request) (*http.Resp
 		return nil, err
 	}
 	level.Info(c.logger).Log("msg", "DoScrape", "scrape_id", id, "url", r.URL.String())
+	// It is important this id is cryptographically generated as it is relied
+	// upon to match the request and the response.
```

Comment:
While technically correct I think this comment belongs a) into a separate commit and b) in the documentation for `Coordinator.genID`. We could also look into adding the client FQDN to the response ID.

Reply:
> belongs a) into a separate commit 

- [ ] Split comment into its own commit

>  b) in the documentation for Coordinator.genID

- [ ] Add comment to Coordinator.genID

> We could also look into adding the client FQDN to the response ID.

what would be the benefit of doing that?

Reply:
> While technically correct I think this comment belongs a) into a separate commit and b) in the documentation for Coordinator.genID

Done, does that look better?

I was not entirely sure on the wording of the comment to make the requirement clear though